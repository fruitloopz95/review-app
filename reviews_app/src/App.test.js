import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer' ;
import App from './App';

test( 'renders correctly' , () => {
const tree = renderer
.create( <h2>The review space </h2> )
.toJSON();
expect(tree).toMatchSnapshot();
});
