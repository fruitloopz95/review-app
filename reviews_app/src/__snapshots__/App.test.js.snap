// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2>
  The review space 
</h2>
`;
